Extensions/messaging_tweaks.js

```diff
@@ -47,6 +57,26 @@ XKit.extensions.messaging_tweaks = new Object({
 		},
 		
 		"sep2": {
+			text: "Notification Tweaks",
+			type: "separator",
+		},
+		"sound_notification": {
+			text: "Make a sound if you receive a new message",
+			default: true,
+			value: true
```

Comment:
Did that because the sounds in XIM back then where also automatically enabled. Changed though, gonna commit in a moment